### DISCUSSION TITLE
Lazyload overriding

### DISCUSSION THREAD

**Discussion by mtxadmin** [uBlockOrigin/uAssets#18851] [Q&A] [answered]
Is there a way to override lazyload mechanism and load pictures even in case of script errors?

For instance, tomshardware.com.
If we block all trackers there:
scorecardresearch.com
taboola.com
dotmetrics.net
google-analytics.com
hawk.tomshardware.com
permutive.app
privacy-mgmt.com
then js on the page will broken, and lazyload pictures will not show.

Can we forcely replace data-srcset to src attribute in img tag?

There are a lot of such lazyloaded sites nowadays, thg taken only as example.

**Comment by uBlock-user** [comment]
> Can we forcely replace data-srcset to src attribute in img tag?

Not possible without renaming the attribute.

**Reply by MasterKia** [reply, marked as answer]
https://github.com/uBlockOrigin/uBlock-issues/issues/2347 ?

**Reply by mtxadmin** [reply]
Wow! Thank you!

**Comment by gorhill** [comment]
> lazyload pictures will not show

I cannot reproduce such issue with default lists, and I also cannot reproduce when hard-blocking `hawk.tomshardware.com` and `privacy-mgmt.com`. I don't see `taboola.com` being reported in the popup panel.

**Reply by stephenhawk8054** [reply]
The only way I can reproduce is blocking `vanilla.futurecdn.net` scripts

```
tomshardware.com https://vanilla.futurecdn.net script block
```
